org.eclipse.wildwebdeveloper.xml/src/org/eclipse/wildwebdeveloper/xml/XMLLanguageServer.java

```diff
@@ -47,6 +50,35 @@ public XMLLanguageServer() {
 		}
 	}
 
+	/**
+	 * Returns a string of colon (or semi-colon on Windows) separated XML extension
+	 * jar paths.
+	 *
+	 * @return The extension jar paths separated by a colon (or semi-colon on
+	 *         Windows), or an empty string if no extension jars are contributed.
+	 * @throws IOException
+	 */
+	private String getExtensionJarPaths() throws IOException {
+		XMLExtensionRegistry extensionJarRegistry = new XMLExtensionRegistry();
+		List<String> extensionJars = extensionJarRegistry.getXMLExtensionJars();
+		List<String> extensionJarsPaths = new ArrayList<>();
+		if (extensionJars.isEmpty()) {
+			return "";
+		}
+		for (String relativePath : extensionJars) {
+			URL url = FileLocator
+					.toFileURL(FileLocator.find(Platform.getBundle(Activator.PLUGIN_ID), new Path(relativePath)));
```

Comment:
This will in most cases be wrong: the path is relative to the contributing plugin id, not to the org.eclipse.wildwebdeveloper.xml bundle.